cloud-consumption/src/test/java/com/sequenceiq/consumption/converter/CredentialResponseToCloudCredentialConverterTest.java

```diff
@@ -0,0 +1,82 @@
+package com.sequenceiq.consumption.converter;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import com.sequenceiq.cloudbreak.cloud.model.CloudCredential;
+import com.sequenceiq.cloudbreak.service.secret.model.SecretResponse;
+import com.sequenceiq.cloudbreak.service.secret.service.SecretService;
+import com.sequenceiq.environment.api.v1.credential.model.response.CredentialResponse;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+@ExtendWith(MockitoExtension.class)
+public class CredentialResponseToCloudCredentialConverterTest {
```

Comment:
JUnit 5 test class and methods are not needed to be `public`

Reply:
I see, thanks.